### DISCUSSION TITLE
Importing pySCENIC GRN to SCENIC+

### DISCUSSION THREAD

**Discussion by Tripfantasy** [aertslab/scenicplus#243] [Q&A]
Hello, 

Is it possible to run GRN steps in pySCENIC on scRNA-seq data , and transfer over the information for use in the SCENIC+ pipeline at some point to avoid the ATAC requirement steps? Or would it make more sense to just use pySCENIC / try to outsource similar ATAC data? 

Thanks in advance!

**Comment by SeppeDeWinter** [comment]
Hi @Tripfantasy 

To be able to GRN inference run SCENIC+ you need scATAC-seq data.

What aspects of SCENIC+ would you like to use?

In theory it is possible to use scATAC-seq data that was not generated by you, as long as the cell types present in both the scRNA-seq and the scATAC-seq are the same/very similar to a certain extend.


Best,

Seppe
